Dockerfile

```diff
@@ -98,6 +98,7 @@ COPY --from=argoexec-build /usr/local/bin/kubectl /bin/
 COPY --from=argoexec-build /usr/local/bin/jq /bin/
```

Comment:
@alexec I'm not sure why but the image is the root cause.

Reply:
https://www.reddit.com/r/docker/comments/8y2zyx/how_to_add_a_empty_directory_to_a_scratch_image/hneaaqe/